backend/src/test/java/org/cryptomator/hub/api/DeviceResourceIT.java

```diff
@@ -293,12 +350,24 @@ public void testGetSomeNotExisting() {
 		}
 
 		@Test
-		@DisplayName("GET /devices?ids=device2&ids=device3 returns 200 with body containing device2 and device3")
+		@DisplayName("GET /devices?ids=device2&ids=device3&ids=legacyDevice1 returns 200 with body containing device2 and device3")
```

Comment:
```suggestion
		@DisplayName("GET /devices?ids=device2&ids=device3&ids=legacyDevice1 returns 200 with body containing device2, device3 and legacyDevice1")
```

Reply:
No, `legacyDevice1` is not returned, because it is a legacy device, see https://github.com/cryptomator/hub/pull/331/files/dcd175a8b3e5360a24408fdefdb0fbd3f573f039#diff-67652a0e51b56759c5824de898118a1bdb5cbf03d18b39d6aa77a54a5b38c32bR358